src/main/java/uk/gov/justice/hmpps/prison/api/resource/LocationResource.java

```diff
@@ -95,8 +97,17 @@ public LocationResource(final AuthenticationFacade authenticationFacade, final L
             @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class)})
     @ApiOperation(value = "Location detail.", notes = "Location detail.", nickname = "getLocation")
     @GetMapping("/{locationId}")
-    public Location getLocation(@PathVariable("locationId") @ApiParam(value = "The location id of location", required = true) final Long locationId) {
-        return locationService.getLocation(locationId);
+    public Location getLocation(
+        @PathVariable("locationId")
+        @ApiParam(value = "The location id of location", required = true)
+        final Long locationId,
+
+        @RequestParam(value="includeInactive", required = false)
+        @Pattern(regexp="Yes", flags = {Flag.CASE_INSENSITIVE})
```

Comment:
An interesting approach!  In other places we have used booleans...  I wonder if we should do the same for consistency?

Reply:
Replaced with Boolean